backend/src/appointment/l10n.py

```diff
@@ -1,10 +1,18 @@
 from typing import Union, Dict, Any
 
 from starlette_context import context, errors
+from .middleware.l10n import get_fluent
 
 
-def l10n(msg_id: str, args: Union[Dict[str, Any], None] = None) -> str:
-    """Helper function to automatically call fluent.format_value from context"""
+def l10n(msg_id: str, args: Union[Dict[str, Any], None] = None, lang: str = None) -> str:
```

Comment:
The l10n helper now accepts a third parameter for a custom language, independent from the request.